manifests/server.pp

```diff
@@ -574,9 +580,28 @@
         ensure   => running,
         enable   => true,
         provider => 'systemd',
-        require  => [ File["/etc/openvpn/${name}.conf"], Openvpn::Ca[$ca_name] ]
+        require  => [ File["${etc_directory}/openvpn/${name}.conf"], Openvpn::Ca[$ca_name] ]
       }
     }
   }
 
+  if $::openvpn::params::namespecific_rclink {
```

Comment:
Fixed in https://github.com/dennisdegreef/puppet-openvpn/commit/ed100549bb4fac3a10fff22984f9396a0bce8682